tabbycat/draw/views.py

```diff
@@ -828,6 +835,7 @@ def get_table(self):
 
 
 class SideAllocationsView(AdministratorMixin, BaseSideAllocationsView):
+    edit_permission = Permission.EDIT_ALLOCATESIDES
```

Comment:
Not defined.

Also, maybe it should become a `view_permission` as the permissions aren't hooked up to Vue pages, and there'd be no need to get to the page if not to edit. Same thing for Adj, Preformed Panel, and Venue allocation.

Reply:
I am pretty sure that it is defined in `permissions.py`, but yes; I will change to `view_permission`.

Reply:
Fair, must just have been hidden by the diff